### ISSUE TITLE
Reorganize the Cloud deployment section and introductions

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-platform-guide#593]
The currently deployed guide has a bit of a mix on the Cloud Deployment section:

<img width="313" alt="Screenshot 2021-02-25 at 12 10 31" src="https://user-images.githubusercontent.com/762126/109145285-97ab0500-7762-11eb-9fec-f04a39e55645.png">

A more convenient/usable distribution of the information could be:

- (some intro here?)
- AWS
	- EKS
	- RDS
	- Ingress thingies
	- TLS
	- another AWS-specific tech
	- ...
- GCP 
	- GCP k8s cluster
	- GCP Cloud SQL
	- Ingress thingies
	- TLS
	- another GCP-specific tech
	- ...
- Managed Services
	- Kafka - Confluent Cloud
	- Kafka - Strimzi Operator
	- Datastax Astra (?)
	- other managed services
- Akka Operator Reference
- Troubleshooting

**Comment by ignasi35** [comment]
PS: I tried to follow the exact same list on the AWS and GCP sections:

- cluster
- SQL
- ingress
- TLS
- others...

**Comment by ignasi35** [comment]
ping @thothmoses

**Comment by thothmoses** [comment]
@ignasi35, @rstento I took a swipe at outlining this so we can begin rolling forward on it. This should happen in parallel with the creation of the additional GPC materials.
https://docs.google.com/document/d/1Jh9VMfrOKkaXgj4EP4NTeTG7KRPXc6Cu24rEz_vWLaQ/edit?usp=sharing

**Comment by patriknw** [comment]
Keep me in the loop of this. I'm not sure I understand the suggestions in the document.

**Comment by rstento** [comment]
@thothmoses, nice job! It took me a while to grok what you were suggesting and I've countered in a number of places. I do believe that re-organizing the whole guide at this point is just too much to bite off. I recommend instead focusing on what are the two most important pieces: Rewritting the first page and reorganizing the Deployment chapter in prep for including GCP content. @patriknw @ignasi35 @jtownley, would you agree? In fact, this issue is titled "Reorganize deployment section" and I think that can be started. Another issue should be created for reorganizing the guide and one for rewriting the Introduction (first page). These related issues can be organized under an epic.

**Comment by rstento** [comment]
Also I wanted to note that as @thothmoses works through reorganizing these pages, it is very likely that he will see ways of fine-tuning, such as renaming pages and headings and shuffling them around.

**Comment by patriknw** [comment]
>  I recommend instead focusing on what are the two most important pieces: Rewritting the first page and reorganizing the Deployment chapter in prep for including GCP content

That sounds very good, and is needed.

**Comment by ignasi35** [comment]
@thothmoses I drafted some changes in https://github.com/akka/akka-platform-guide/pull/598 already. Feel free to take that PR for a spin or create a new one (and we'll close #598). :-)

**Comment by thothmoses** [comment]
Primary restructure complete, Primary GCP content in.
